packages/jellyfish-api-core/__tests__/category/token.test.ts

```diff
@@ -470,20 +470,20 @@ describe('masternode', () => {
   })
 
   describe('getCustomTx', () => {
-    it('should getCustomTx with ', async () => {
-      let txid: any
+    let txid: string
 
-      beforeAll(async () => {
-        txid = await createToken('DSWAP')
-        await container.generate(3)
-      })
+    beforeAll(async () => {
+      txid = await createToken('DSWAP')
+      await container.generate(3)
```

Comment:
- move it into `ln:481` since it just run once and easier to read
- generate(1) is enough